docker-compose.yml

```diff
@@ -5,15 +5,19 @@ web:
     - '443/tcp'
   volumes:
     - './web:/var/www/html'
+    - './drupal.make:/var/www/html/drupal.make'
     - './docker/settings.php:/var/www/html/sites/default/settings.php:ro'
+    - './docker/settings.local.php:/var/www/html/sites/default/settings.local.php:ro'
     - './docker/drupal-cron.conf:/etc/cron.d/drupal-cron.conf:ro'
     - './docker/prepare-environment.sh:/etc/my_init.d/prepare-environment.sh'
     - './docker/php.ini:/etc/php/5.6/mods-available/ding.ini'
+    - "${SSH_AUTH_SOCK}:/tmp/ssh-agent"
```

Comment:
I just tested this from a Mac and got the follow error when runing drush_make from the container:

`Unable to clone ding2 from git@github.com:ding2/ding2.git`.

This corresponds with the following error when exec'ing into bash on the web container and running trying to clone the repo:

```
root@7b2371ff4ef4:/var/www/html# git clone git@github.com:ding2/ding2.git
Cloning into 'ding2'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.